components/inputs/input-date.js

```diff
@@ -349,8 +353,19 @@ class InputDate extends SkeletonMixin(FormElementMixin(LocalizeCoreElement(LitEl
 		this._calendar.focus();
 	}
 
+	_handleInputTextBlur() {
+		this._inputTextFocusMouseup = false;
+		this._inputTextFocusShowTooltip = true;
+	}
+
 	_handleInputTextFocus() {
 		this._formattedValue = this._shownValue ? formatISODateInUserCalDescriptor(this._shownValue) : '';
+
+		// hide tooltip when focus, wait to see if click happened, then show
+		this._inputTextFocusShowTooltip = false;
+		setTimeout(() => {
+			if (!this._inputTextFocusMouseup) this._inputTextFocusShowTooltip = true;
```

Comment:
Crazy that there isn't a better way to detect the source (mouse vs. keyboard) of a focus event but you're right... I couldn't find another way either.